benchmarks/tool/compile_benchmarks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I think we should remove this if we go with the python script.

Reply:
Done.